### ISSUE TITLE
Add Option: use vcs-friendly ,

### ISSUE THREAD

**Issue by simonharrer** [JabRef/jabref#481]
Changelog in git
```
-  File                     = {:filename.pdf:PDF}
+  File                     = {:filename.pdf:PDF},
+  Keywords                 = {important}
 }
```

but should be
```
+  Keywords                 = {important},
 }
```

Solution: each key/value pair ends with a `,`, even the last one. 

Comments?

**Comment by lenhard** [comment]
I'm against offering an option for this. Either we decide to do this as part of #116, or we leave it.

I'm undecided. The change makes sense from my point of view, but I am afraid that the language purists will throw a tantrum.

The change would be terribly easy to implement, though...

**Comment by stefan-kolb** [comment]
I absolutely agree with Jörg.

**Comment by koppor** [comment]
Think, majority of the users is not used of this feature.

**Comment by stefan-kolb** [comment]
Implemented by 8d971e10be4b8ca3097e68c1230f5caa04a84e90
